phoenix-flume-base/pom.xml

```diff
@@ -46,6 +52,11 @@
       <classifier>tests</classifier>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.phoenix</groupId>
+      <artifactId>phoenix${phoenix.compat.version}-compat</artifactId>
+      <version>6.0.0-SNAPSHOT</version>
```

Comment:
Easier to add this to `dependencyManagement` in the parent pom, then omit the `version` in the other poms.